### PR TITLE
added an example for command service

### PR DESCRIPTION
**Pull request by cdietrich**
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
please review

File: vscode-extension/package.json [hunk 1]
```diff
@@ -50,12 +60,12 @@
         "@types/node": "^6.0.88"
     },
     "dependencies": {
-        "vscode-languageclient": "^3.4.1"
+        "vscode-languageclient": "^3.5.0"
```
Comment (kthoms) [inline vscode-extension/package.json]: is the upgrade required for this?

File: vscode-extension/src/extension.ts [hunk 2]
```diff
@@ -3,8 +3,8 @@
 import * as net from 'net';
 
 import {Trace} from 'vscode-jsonrpc';
-import { workspace, ExtensionContext } from 'vscode';
-import { LanguageClient, LanguageClientOptions, StreamInfo } from 'vscode-languageclient';
+import { window,workspace, commands, ExtensionContext, Uri } from 'vscode';
```
Comment (kthoms) [inline vscode-extension/src/extension.ts]: space after window,

File: vscode-extension/package.json (hunk 1 above)
Comment (cdietrich) [reply vscode-extension/package.json]: looks so

File: vscode-extension/src/extension.ts (hunk 2 above)
Comment (cdietrich) [reply vscode-extension/src/extension.ts]: done

**Pull request merged**